prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="file:///Users/jeesupark/repos/prep-90921-code-solutions/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
This will only work on your system, if someone else downloaded your files the links would not work. This is know as an absolute path. For links to different pages we want to use a relative path. Here your `href` would be set to `../index.html`